### PR TITLE
Rename Query History Actions

### PR DESCRIPTION
**Pull request by norascheuch**
The next version of the VS Code extension will rename some actions on the query history menu.

### CODE REVIEW

**Review by koesie10 — APPROVED** [review]
LGTM, but we'll probably need to wait until the extension has been released before merging this.

File: docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst
Comment (felicitymay) [inline docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst:125]: I'm not sure whether this is a typo, or whether both "Open Query" and "Show Query" are now covered by "View Query". If it isn't a typo, then we'll need to change or remove the final sentence here.

**Review by felicitymay — COMMENTED** [review]
I'm not sure of the release schedule for the VS Code extension but agree that this should be merged to ensure that the changes are published at the right moment.

We update the docs for every CodeQL CLI release, publishing from the relevant release candidate branch.

File: docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst
Comment (norascheuch) [reply docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst:125]: I'm not sure I understand @felicitymay. `Open Query` was renamed to `View Query`, `Show Query Text` renamed to `View Query Text`. Am I missing something?

Comment (felicitymay) [reply docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst:125]: My apologies. It seems that I can't read properly. Somehow I read both of those as the same option 🤦🏻‍♀️

**Review by felicitymay — APPROVED** [review]
Thanks for your patience with my misunderstanding @norascheuch.

This looks ready to merge when you have a date for the release.

**Comment by charisk** [comment]
A new release was published on February the 3rd, shall we merge this?

**Comment by felicitymay** [comment]
> A new release was published on February the 3rd (https://github.com/github/vscode-codeql/releases/tag/v1.7.8), shall we merge this?

If you merge this PR the updated articles will be published with the next release of the CodeQL CLI in about two weeks. Is that okay?

**Comment by charisk** [comment]
> > A new release was published on February the 3rd (https://github.com/github/vscode-codeql/releases/tag/v1.7.8), shall we merge this?
> 
> If you merge this PR the updated articles will be published with the next release of the CodeQL CLI in about two weeks. Is that okay?

That sounds good to me. @norascheuch, @koesie10?

**Pull request merged**
